jest.config.js

```diff
@@ -6,4 +7,9 @@ module.exports = {
     '@/([^\\.]*)$': '<rootDir>/src/$1',
   },
   testPathIgnorePatterns: ['<rootDir>/cypress/', '<rootDir>/node_modules/'],
+  globals: {
+    'ts-jest': {
+      isolatedModules: true,
+    },
+  },
```

Comment:
This fixes a jest memory leak when using typescript.

Before:
![Code_w1BKfr265f](https://user-images.githubusercontent.com/13315440/140415393-7659a0f9-0338-4908-a466-c07b87904964.png)

After:
![Code_FXU7EN3REx](https://user-images.githubusercontent.com/13315440/140415410-449c59d8-8785-4cb0-b345-ce966a323834.png)

Reply:
🥇

Reply:
Found the fix in this thread: https://github.com/kulshekhar/ts-jest/issues/1967